Dockerfile

```diff
@@ -1,49 +1,56 @@
 # Forked from TrafeX/docker-php-nginx (https://github.com/TrafeX/docker-php-nginx/)
 
-FROM alpine:latest
+FROM alpine:3.12
 LABEL Maintainer="Aurélien JANVIER <dev@ajanvier.fr>" \
       Description="Unofficial Docker image for Polr."
 
-# Environment variables
-ENV APP_NAME My Polr
-ENV APP_PROTOCOL https://
-ENV DB_PORT 3306
-ENV DB_DATABASE polr
-ENV DB_USERNAME polr
-ENV POLR_BASE 62
-
-# Install packages
-RUN apk --no-cache add gettext git php7 php7-fpm php7-pdo php7-mysqli php7-json php7-openssl php7-curl \
-    php7-zlib php7-xml php7-phar php7-intl php7-dom php7-xmlreader php7-ctype \
-    php7-mbstring php7-gd php7-xmlwriter php7-tokenizer php7-pdo_mysql php7-memcached nginx supervisor curl bash
+# Install packages and remove default server definition
+RUN apk --no-cache add bash git php7 php7-fpm php7-opcache php7-mysqli php7-json php7-openssl php7-curl \
+        php7-zlib php7-xml php-xmlwriter php7-phar php7-intl php7-dom php7-xmlreader php7-ctype php7-session \
+        php7-mbstring php7-gd php7-pdo php7-pdo_mysql php7-tokenizer nginx supervisor curl && \
+    apk add --update libintl && \
+    apk add --virtual build_deps gettext &&  \
+    cp /usr/bin/envsubst /usr/local/bin/envsubst && \
```

Comment:
required to use envsubst on this alpine version.
more info on https://github.com/suzuki-shunsuke/docker-image-alpine-envsubst